module/sampleon/index.js

```diff
@@ -1,7 +1,8 @@
 var flyd = require('../../lib');
+var R = require('ramda');
```

Comment:
Pulling in the entire ramda library as a dependency seems kind of unecessary here, since you only use it for `R.contains`. You might saw the ramda dependency in `package.json` but only `ramda/src/curryN` is actually used in the flyd source. This would increase the bundle size by quite a bunch.